src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Switch, Route, Link,
+} from 'react-router-dom';
+import Books from './components/Books';
+import Categories from './components/Categories';
+
+const App = () => (
+  <div>
+    <h1>Welcome to BookStore</h1>
+    <Router>
+      <Link to="/books">Books</Link>
+      <Link to="/categories">Categories</Link>
+      <Switch>
+        <Route path="/books">
```

Comment:
* Kindly create a component that Renders a form with inputs for the title and the author of the book with a submit button but the button isn't expected to work for now just needs to be rendered
* Refer to project sneak peak 
![image](https://user-images.githubusercontent.com/52497246/130642369-5148d970-6ade-42fa-bec7-49c68f8482d7.png)